pkg/minikube/download/binary.go

```diff
@@ -46,13 +46,20 @@ func binaryWithChecksumURL(binaryName, version, osName, archName string) (string
 func Binary(binary, version, osName, archName string) (string, error) {
 	targetDir := localpath.MakeMiniPath("cache", osName, version)
 	targetFilepath := path.Join(targetDir, binary)
+	targetLock := targetFilepath + ".lock"
 
 	url, err := binaryWithChecksumURL(binary, version, osName, archName)
 	if err != nil {
 		return "", err
 	}
 
-	if _, err := os.Stat(targetFilepath); err == nil {
+	releaser, err := lockDownload(targetLock)
+	defer releaser.Release()
```

Comment:
releaser can be nil. (Line #115 of `lockDownload`)

Reply:
Fixed.